src/arith_uint256.h

```diff
@@ -33,21 +33,21 @@ class base_uint
     {
         static_assert(BITS/32 > 0 && BITS%32 == 0, "Template parameter BITS must be a positive multiple of 32.");
 
-        for (int i = 0; i < WIDTH; i++)
```

Comment:
While you're touching these, can you add braces too (in a separate commit in the same PR)?